CHANGELOG.rst

```diff
@@ -58,7 +58,7 @@ New features
 * New flag ``--porcelain`` for programmatic integrations to use. See the
   ``integrations`` section :doc:`here </usage>` for details.
 * Implement a new :doc:`configuration option </configure>`: ``default_due``.
-* The configuration file is now pre-emptively validated. Users will be warned
```

Comment:
Actually, my dictionary suggests the opposite correction. I quickly looked this up, and the correct spelling has a dash ([source](https://en.oxforddictionaries.com/definition/pre-emptively)).